JoinColony/colonyDapp
# Add RemainingDisplayWidget component

Added new RemainingDisplayWidget component to CoinMachine in order to display time and tokens remaining

**New stuff** ✨

* Added `question-mark-inverted` icon
* Added `invertedIcon` prop to QuestionMarkTooltip component to flip the icon's color
* Added `RemainingDisplayWidget` to CoinMachine
* Added `TokenPriceStatusIcon` component to `RemainingDisplayWidget`

### How to test each case
1. The first case is the one with placeholders so basically just enter the coin machine page
2. The second case is the one where values are added so just add some string like `0:00:00` or `1200/2500` to the `value` prop of the widget. Also, in the case of the tokens remaining footer, in order to see the price status icon, you need to pass a `TokenPriceStatuses` value to the widget with a `footerText`.
3. The third case is the one with red background and you just need to pass an `appearance={{ theme: 'danger' }}` to the widget.
4. The final case is when the values are displayed in red, just change the `false` to `true` on line 64 of the RemainingDisplayWidget.

![FireShot Capture 391 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646297-e86d3299-46cc-45a7-af9e-a6f985bcb73b.png)

![FireShot Capture 392 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646304-0d8aef33-86ab-452c-a3e5-c610c8ca3635.png)

![FireShot Capture 393 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646312-14e69c3f-7489-4a55-8322-8fa0b8b88e56.png)

![FireShot Capture 394 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646315-3f3002d2-eb5c-45cb-9996-f359a35cfe9b.png)

![FireShot Capture 395 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646321-888c3091-75fe-4dc5-9bd8-69aecd843756.png)

![FireShot Capture 396 - Colony - localhost](https://user-images.githubusercontent.com/18473896/126646326-5c90879c-0911-494c-a2ac-a33772d6ca53.png)

Resolves DEV-453
